src/settled-physical/CrossMarginPhysicalMath.sol

```diff
@@ -150,7 +150,7 @@ library CrossMarginPhysicalMath {
             }
         }
 
-        for (i; i < _detail.callWeights.length;) {
+        for (i = 0; i < _detail.callWeights.length;) {
```

Comment:
Looks like this is also an issue in `CrossMarginCashMath.sol`

Reply:
added it in with the last commit, thanks